package.json

```diff
@@ -20,7 +20,10 @@
     "nodemon": "^1.19.1"
   },
   "dependencies": {
+    "axios": "^0.19.0",
```

Comment:
don't think you'll need axios in the backend. You can use 'yarn remove' to uninstall it from your yarn.lock and package.json file